docs/windows.md

```diff
@@ -207,8 +207,16 @@ kubectl apply -f -
 ```
 
 Since Antrea 1.13, you can deploy both antrea-agent and antrea-ovs Windows DaemonSets
-with containerd runtime by applying file `antrea-windows-containerd-with-ovs.yml`. The
-following commands download the manifest, set kubeAPIServerOverride, and deploy
+with containerd runtime by applying file `antrea-windows-containerd-with-ovs.yml`.If the
+user wants to use an unsigned OVS kernel driver, it must be pre-installed on the
+host before applying the `antrea-windows-containerd-with-ovs.yml` file
+using the `Install-OVS.ps1` script as below:
```

Comment:
what does the "host" refer to here? Every worker Node in the cluster?

These are the steps a user would follow:
1) apply `antrea-windows-containerd-with-ovs.yml`
2) prepare windows worker Node (that includes installing OVS if needed)
3) add windows worker Node

So the following instruction:

> If the user wants to use an unsigned OVS kernel driver, it must be pre-installed on the host before applying the `antrea-windows-containerd-with-ovs.yml` file

doesn't make sense to me.

-------

This is what the documentation should look like IMO:
- remove the added content from this section (just ask the user to apply `antrea-windows-containerd-with-ovs.yml`)
- modify section `1. (Optional) Install OVS (provided by Antrea or your own)` below to add the necessary information. User will always apply the manifest first, but they need to _prepare_ Windows Nodes correctly before _joining_ them to the cluster.